deeppavlov/models/ranking/bilstm_network.py

```diff
@@ -0,0 +1,279 @@
+import numpy as np
+from typing import List
+from keras.layers import Input, LSTM, Embedding, GlobalMaxPooling1D, Lambda, Dense, Layer
+from keras.layers.merge import Multiply
+from keras.models import Model
+from keras.layers.wrappers import Bidirectional
+from keras.optimizers import Adam
+from keras.initializers import glorot_uniform, Orthogonal
+from keras import losses
+from keras import backend as K
+from tensorflow.python.framework.ops import Tensor
+
+from deeppavlov.core.common.log import get_logger
+from deeppavlov.core.common.registry import register
+from deeppavlov.models.ranking.keras_siamese_model import KerasSiameseModel
+
+log = get_logger(__name__)
+
+
+@register('bilstm_nn')
+class BiLSTMNetwork(KerasSiameseModel):
```

Comment:
Don't you think this name is too generic? What about BiLSTMNetwork for sequence tagging or for simple classification that people potentially could contribute?

Reply:
renamed to BiLSTMSiameseNetwork